cognite/powerops/resync/models/v2/dms/data_models/views/dayAheadBids-SHOPPriceScenarioResult.view.yaml

```diff
@@ -0,0 +1,34 @@
+space: '{{dayahead_model_space}}'
+externalId: SHOPPriceScenarioResult
```

Comment:
Ref. the comment above (about SHOPPriceScenario), maybe we should rename this (the SHOPPriceScenarioResult) - either to SHOPScenarioResult (con: may look like a broader result object), or PriceProductionPair (con: does not "cover" that we refer to a SHOP Scenario), or SHOPPriceAndProductionResult